### PR TITLE
LB-1689: Improve last.fm loved tracks import

### PR DESCRIPTION
**Pull request by amCap1712**
The existing importer only tries to import tracks with track_mbids. Extend it to recognise recording_mbids and for tracks without mbids do a messybrainz lookup.

### CODE REVIEW

File: listenbrainz/domain/lastfm.py [hunk 1]
```diff
@@ -88,27 +89,43 @@ def fetch_lfm_feedback(lfm_user: str):
...
+            except (ValueError, TypeError):
+                item["mbid"] = None
 
-    return track_mbids, track_data, counts
+            items.append(item)
 
+    return items, total_count
+
+
+def bulk_get_msids(connection, items):
```
Comment (mayhem) [inline listenbrainz/domain/lastfm.py:109]: docstring

File: listenbrainz/domain/musicbrainz.py [hunk 2]
```diff
@@ -41,3 +45,46 @@ def get_user_info(self, token: str):
         )
         response.raise_for_status()
         return response.json()
+
+    def _submit_review_to_CB(self, token: str, review: CBReviewMetadata):
```
Comment (mayhem) [inline listenbrainz/domain/musicbrainz.py:49]: This seem unrelated...

**Review by mayhem — APPROVED** [review]
Minor cleanup, after that good to go

File: listenbrainz/domain/lastfm.py (hunk 1 above)
Comment (amCap1712) [reply listenbrainz/domain/lastfm.py:109]: fixed.

File: listenbrainz/domain/musicbrainz.py (hunk 2 above)
Comment (amCap1712) [reply listenbrainz/domain/musicbrainz.py]: my bad. removed.
